### ISSUE TITLE
Feedback for “Introduction”

### ISSUE THREAD

**Issue by oddjob23** [bigcommerce/dev-docs#1980]
Following page has an error: https://developer.bigcommerce.com/api-docs/apps/tutorials/sample-app-nextjs/step-1-setup

It has TypeError: Cannot read properties of undefined (reading 'apiOrigin')

**Comment by slsriehl** [comment]
Thanks for reporting, @oddjob23. I've shared this with the engineering team to determine whether it's a docs problem or a code problem.

**Comment by matthewvolk** [comment]
Hi @oddjob23 👋🙂 Thanks for reaching out!

Is there any chance you can share more of the stack trace containing that error you posted? Or, even better, some code samples showing how you're defining that `apiOrigin` variable? I'm unable to find a reference to that identifier anywhere in the sample-app-nodejs repository or dependencies. Seeing a code sample showing how you're using that variable would go a long way in seeing if we can figure this one out with you.

Thanks in advance!

**Comment by oddjob23** [comment]
Hi @matthewvolk! Sorry for not providing more context to this issue. The issue occurred on a documentation page. I was going through the tutorial for the sample nextjs application. When I tried to navigate with the link on the page that leads to the step 1, this one [https://developer.bigcommerce.com/api-docs/apps/tutorials/sample-app-nextjs/step-1-setup](url), page crashed. In the console I found the type error and I copied the outcome. I clicked on the link "Question? Give us feedback" and that's how I opened this issue.

Unfortunately, I don't have more information and if you try to visit this page now, it throws 404. It's not related to any code sample that I was running, just a documentation page that I was visiting.

I hope this was helpful.
